src/com/operators_loops/Factorial.java

```diff
@@ -0,0 +1,20 @@
+package com.operators_loops;
+import java.util.Scanner;
+
+public class Factorial {
+
+    private static final String ENTER_AN_INTEGER = "Enter an integer: ";
+    private static final String FACTORIAL_OF_THE_NUMBER = "Factorial of the number ";
+    private static final String IS = " is ";
+
+    public static void main(String[] args) {
+        System.out.println(ENTER_AN_INTEGER);
+        Scanner scanner = new Scanner(System.in);
+        int number = scanner.nextInt();
+        int factorial = 1;
+        for (int i = 1; i <= number; i++) {
+            factorial = factorial * i;
+        }
+        System.out.println(FACTORIAL_OF_THE_NUMBER + number + IS + factorial);
```

Comment:
в циклах краще нумерувати з 0, так як нумерація майже всього в java починається з 0
```suggestion
        for (int i = 0; i < number; i++) {
            factorial = factorial * i;
        }
        System.out.println(FACTORIAL_OF_THE_NUMBER + number + IS + factorial);
```